src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/storm/bolt/isl/IslHandler.java

```diff
@@ -182,6 +200,11 @@ private Values makeStatusUpdateTuple(IslStatusUpdateNotification payload) {
         return new Values(null, payload, getCommandContext());
     }
 
+    private Values makeWatchListBfdSlowDiscoveryTuple(Endpoint endpoint, boolean enableSlowDiscovery) {
```

Comment:
... incorrect pattern usage. The `makeSomethingTuple` provide API to format tuple for specific stream. These methods are responsible for correct fields ordering and correct value types into produced tuples.

You don't have a "watcher-list-bfd-slow-dicovery" stream.

Reply:
Fixed